fuzzing/frames/fuzz.go

```diff
@@ -41,15 +42,22 @@ func Fuzz(data []byte) int {
 	var b []byte
 	for len(data) > 0 {
 		initialLen := len(data)
-		l, f, err := parser.ParseNext(data, encLevel, version)
+		frameType, l, err := parser.ParseType(data, encLevel)
 		if err != nil {
+			if err == io.EOF { // PADDING frame
```

Comment:
`EOF` doesn't mean PADDING frame, it means we've reached the end of the slice!

```suggestion
			if err == io.EOF {
```

Reply:
Actually, it does, so this comment might be more helpful.
```suggestion
			if err == io.EOF { // the last frame was a PADDING frame
```